### PR TITLE
[Docs] Replace `calicoctl` typos

### PR DESCRIPTION
**Pull request by michaelact**
Product Version(s):

```
# calicoctl version
Client Version:    v3.23.3
Git commit:        3a3559be1
Cluster Version:   v3.23.3
Cluster Type:      kubespray,bgp,kubeadm,kdd
```

Issue:

```
# kubectl get felixconfiguration.p default
error: the server doesn't have a resource type "felixconfiguration"
```

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico_versioned_docs/version-3.26/operations/ebpf/enabling-ebpf.mdx [hunk 1]
```diff
@@ -296,7 +296,7 @@ kubectl patch networks.operator.openshift.io cluster --type merge -p '{"spec":{"
 If you cannot disable `kube-proxy` (for example, because it is managed by your Kubernetes distribution), then you _must_ change Felix configuration parameter `BPFKubeProxyIptablesCleanupEnabled` to `false`. This can be done with `kubectl` as follows:
 
 ```
-kubectl patch felixconfiguration.p default --patch='{"spec": {"bpfKubeProxyIptablesCleanupEnabled": false}}'
+calicoctl patch felixconfiguration default --patch='{"spec": {"bpfKubeProxyIptablesCleanupEnabled": false}}'
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.26/operations/ebpf/enabling-ebpf.mdx]:
```suggestion
kubectl patch felixconfiguration default --patch='{"spec": {"bpfKubeProxyIptablesCleanupEnabled": false}}'



```

**Review by ctauchen — COMMENTED** [review]
Thanks for catching this! We put in a fix in #677 but this unfortunately omitted the 3.26 version of this file.

Generally, we try to use kubectl wherever possible. But the `.p` definitely needed to go.

File: calico_versioned_docs/version-3.26/operations/ebpf/enabling-ebpf.mdx (hunk 1 above)
Comment (michaelact) [reply calico_versioned_docs/version-3.26/operations/ebpf/enabling-ebpf.mdx]: The last time I executed this command, it returned the following error:
```
# kubectl get felixconfiguration.p default
error: the server doesn't have a resource type "felixconfiguration"
```

Comment (ctauchen) [reply calico_versioned_docs/version-3.26/operations/ebpf/enabling-ebpf.mdx]: @lwr20 Can you weigh in here? Is this indeed one that kubectl can't handle? Are the related fixes in #677 wrong?

Comment (lwr20) [reply calico_versioned_docs/version-3.26/operations/ebpf/enabling-ebpf.mdx]: On my kind cluster with calico installed:
```
$ kubectl get felixconfiguration.p default
error: the server doesn't have a resource type "felixconfiguration"
$ kubectl get felixconfiguration default
NAME      AGE
default   3m18s
$ kubectl patch felixconfiguration default --patch='{"spec": {"bpfKubeProxyIptablesCleanupEnabled": false}}'
felixconfiguration.projectcalico.org/default patched
```

Comment (michaelact) [reply calico_versioned_docs/version-3.26/operations/ebpf/enabling-ebpf.mdx]: Maybe the issue was `.p`. I will accept your commit suggestion, @ctauchen.

**Comment by ctauchen** [comment]
@michaelact Thanks! Can you sign the contributors agreement so we can get this merged? https://cla-assistant.io/tigera/docs?pullRequest=742

**Comment by bmckercher123** [comment]
@michaelact Ready to merge. Please sign contributor agreement: https://cla-assistant.io/tigera/docs?pullRequest=742.

**Comment by bmckercher123** [comment]
Closing out as fixes were made here: https://github.com/tigera/docs/pull/677.

**Pull request closed without merging**